<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUVWidgetBase</class>
 <widget class="QWidget" name="QUVWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="_uvFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLogTable" name="_transitionTable" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="numRows" stdset="0">
      <number>3</number>
     </property>
     <property name="numCols" stdset="0">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Show transition dipoles</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="_electricDipoleCheckBox">
        <property name="text">
         <string>Electric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_magneticDipoleCheckBox">
        <property name="text">
         <string>Magnetic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_velocityDipoleCheckBox">
        <property name="text">
         <string>Velocity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="_densityChangesCheckBox">
       <property name="text">
        <string>Show Density Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="_transitionChangesCheckBox">
     <property name="text">
      <string>Show Transition Changes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="_transitionCoefficientsCheckBox">
     <property name="text">
      <string>Show Transition Vector Coefficients</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>UV / ECD Spectrum</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="_spectrumTypeComboBox">
        <item>
         <property name="text">
          <string>UV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ECD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QPushButton" name="_enantiomerButton">
        <property name="text">
         <string>Enantiomer</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="_ECDFormalismComboBox"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="_solventShiftCheckBox">
        <property name="text">
         <string>Substract solvent shifts</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="_rightLimitLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="_leftLimitLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="_resetLimitsButton">
          <property name="text">
           <string>Reset Limits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_exportIRButton">
          <property name="text">
           <string>Write JCAMP-DX</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="showSpecButton">
          <property name="text">
           <string>Show Spectrum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_copyButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy data in a tab-separated format to the clipboard. Data can then be  pasted in a spreadsheet to produce custom charts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Copy data to clipboard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="_numpoints">
          <property name="minimum">
           <number>16</number>
          </property>
          <property name="maximum">
           <number>65536</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Points</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="_shift"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Shift (nm)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="3">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QDoubleSpinBox" name="_lineWidthSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>4.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Line Width (eV)</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QLogTable</class>
   <extends>QWidget</extends>
   <header>qlogtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
